backend/src/main/java/de/neuefische/backend/model/Market.java

```diff
@@ -0,0 +1,24 @@
+package de.neuefische.backend.model;
+
+import lombok.*;
+import org.springframework.data.annotation.Id;
+
+import java.math.BigDecimal;
+import java.util.List;
+
+@AllArgsConstructor
+@RequiredArgsConstructor
+@Builder
+@Data
+public class Market {
+    @Id
+    private String marketId;
```

Comment:
Best practice here is not to say marketId. Id is more than enough. While reading other read somethink like this market.marketId instead of market.id :-)
```suggestion
    private String id;
```